app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for person_data in people:
+        person = Person(person_data["name"], person_data["age"])
+        person_list.append(person)
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
+        if "wife" in person_data and person_data["wife"]:
+            person.wife = Person.people[person_data["wife"]]
```

Comment:
According to the checklist, you should use descriptive and correct variable names. The `person_data` dictionary might contain not only `wife` but also `husband`. So, it would be more appropriate to use a more general term, like `spouse`.